### ISSUE TITLE
PHP 8.0: Union type "TYPE|null" results in "Use of unknown class: 'null'"

### ISSUE THREAD

**Issue by menturion** [DEVSENSE/phptools-docs#342]
Version: `v1.34.13249`

For PHP 8.0, the union type "TYPE|null" results in "`Use of unknown class: 'null'`".
For PHP 8.2 it works fine.

**Comment by jakubmisek** [comment]
may I ask for a test case, it seems to work:

![image](https://github.com/DEVSENSE/phptools-docs/assets/842150/cdc17747-5b86-42d6-80e8-394ae93cecbe)

**Comment by jakubmisek** [comment]
got it, nevermind :) (it reports the warning only when not in a namespace...)

![image](https://github.com/DEVSENSE/phptools-docs/assets/842150/4bf19777-b574-46cc-b9b0-083bac66648b)

**Comment by jakubmisek** [comment]
fixing, thank you!

**Comment by jakubmisek** [comment]
fixed in the next pre-release/release. Thank you!

**Comment by menturion** [comment]
Hi @jakubmisek, 

this issue does not seem to be completely fixed yet.

I get several warnings that "`Return value of myMethod() is expected to be of type NAMESPACE\null|array|bool, null returned`" in case of ...

```
public function myMethod(...): bool|null|array
{
    ...
    return null;
    ...
}
```

No warnings are reported anymore if I change the order of return types to "`public function myMethod(...): bool|array|null`".

**Comment by jakubmisek** [comment]
@menturion you're right, thank you. It only allows `null` to be the last type name (in PHP 8). Will be fixed.

**Comment by jakubmisek** [comment]
the latest pre-release fixes that, thanks
